src/components/book.jsx

```diff
@@ -1,19 +1,44 @@
 import PropTypes from 'prop-types';
+import { FaCircleNotch } from 'react-icons/fa';
 
 function Book({ bookTitle, author, onClick }) {
   return (
-    <div>
-      <div>
-        <p>Action</p>
-        <h2>{bookTitle}</h2>
-        <p>{author}</p>
+    <div className="book-container">
+      <div className="book-info">
+        <p className="action">Action</p>
+        <h2 className="book-title">{bookTitle}</h2>
+        <p className="author">{author}</p>
         <div>
-          <button type="button" onClick={onClick}>
+          <button type="button" className="button comment">
+            comments
+          </button>
+          <button type="button" className="button" onClick={onClick}>
             remove
           </button>
+          <button type="button" className="button">
+            edit
+          </button>
+        </div>
+      </div>
+      <div className="book-state">
+        <div className="percentage">
+          <FaCircleNotch className="fa" size="4em" />
+          <div>
+            <p>
+              89%
+              <br />
+              completed
+            </p>
+          </div>
+        </div>
+        <div className="chapter-div">
+          <p className="chapter current">CURRENT CHAPTER</p>
+          <p className="chapter number">Chapter 17</p>
+          <button type="button" className="update">
+            update progress
+          </button>
```

Comment:
- [ ] I think you should increase the padding around this button to match the zeplin dessign 💯 

your design 👇🏽 
![image](https://github.com/mershark/bookstore/assets/63473584/6ecd04fb-daa3-4491-8f42-550806178c95)

zeplin design 👇🏽 
![image](https://github.com/mershark/bookstore/assets/63473584/27faa471-ae66-40bb-9cd0-132a12bd497f)